### PR TITLE
[ja] Translate tasks/administer-cluster/securing-a-cluster into Japanese

### LINKED ISSUE #30810: [ja] Translate tasks/administer-cluster/securing-a-cluster into Japanese
# This is a Feature Request

## What would you like to be added

Translate [tasks/administer-cluster/securing-a-cluste](https://kubernetes.io/docs/tasks/administer-cluster/securing-a-cluster/)  into Japanese.

## Why is this needed

This page has not been translated.

## Comments

/language ja
/assign

### PR DESCRIPTION
**Pull request by ptux**
resolve #30810

### CODE REVIEW

File: content/ja/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
+---
+title: クラスターのセキュリティ
+content_type: task
+---
+
+<!-- overview -->
+
+このドキュメントでは、偶発的または悪意のあるアクセスからクラスタを保護するためのトピックについて説明します。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:8]:
```suggestion
このドキュメントでは、偶発的または悪意のあるアクセスからクラスターを保護するためのトピックについて説明します。
```

Comment (nasa9084) [reply content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:8]: 以下同様に修正お願いします

File: content/ja/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+
+
+
+<!-- steps -->
+
+## Kubernetes APIへのアクセスの制御
+
+Kubernetesは完全にAPI駆動であるため、誰がクラスタにアクセスできるか、どのようなアクションを実行できるかを制御・制限することが第一の防御策となります。
+
+### すべてのAPIトラフィックにTLS（Transport Layer Security）の使用
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:25]:
```suggestion
### すべてのAPIトラフィックにTLS(Transport Layer Security)の使用
```

File: content/ja/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+
+### APIの認可
+
+認証されると、すべてのAPIコールは認可チェックを通過することになります。
+
+Kubernetesには、統合された[Role-Based Access Control (RBAC)](/docs/reference/access-authn-authz/rbac/)コンポーネントが搭載されており、入力されたユーザーやグループを、ロールにまとめられたパーミッションのセットにマッチさせます。
+これらのパーミッションは、動詞(get, create, delete)とリソース(pods, services, nodes)を組み合わせたもので、ネームスペース・スコープまたはクラスター・スコープに対応しています。
+すぐに使えるロールのセットが提供されており、クライアントが実行したいアクションに応じて、デフォルトで適切な責任の分離を提供します。
+
+It is recommended that you use the [Node](/docs/reference/access-authn-authz/node/) and [RBAC](/docs/reference/access-authn-authz/rbac/) authorizers together, in combination with the [NodeRestriction](/docs/reference/access-authn-authz/admission-controllers/#noderestriction) admission plugin.
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:50]: 消し忘れ

File: content/ja/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+
+[Node](/docs/reference/access-authn-authz/node/)と[RBAC](/docs/reference/access-authn-authz/rbac/)の承認者は、[NodeRestriction](/docs/reference/access-authn-authz/admission-controllers/#noderestriction)のアドミッションプラグインと組み合わせて使用することをお勧めします。
+
+認証の場合と同様に、小規模なクラスタにはシンプルで幅広い役割が適切かもしれません。
+しかし、より多くのユーザーがクラスターに関わるようになるとチームを別の名前空間に分け、より限定的な役割を持たせることが必要になるかもしれません。
+認可においては、あるオブジェクトの更新が、他の場所でどのようなアクションを起こすかを理解することが重要です。
+
+たとえば、ユーザーは直接ポッドを作成することはできませんが、ユーザーに代わってポッドを作成するデプロイメントの作成を許可することで、間接的にそれらのポッドを作成することができます。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:58]:
```suggestion
たとえば、ユーザーは直接Podを作成することはできませんが、ユーザーに代わってPodを作成するDeploymentの作成を許可することで、間接的にそれらのPodを作成することができます。
```

Comment (nasa9084) [reply content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:58]: 以下同様

File: content/ja/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+
+[Resource quota](/docs/concepts/policy/resource-quota/)は、ネームスペースに付与されるリソースの数や容量を制限するものです。
+
+これは、ネームスペースが割り当てることのできるCPU、メモリ、永続的なディスクの量を制限するためによく使われますが、各ネームスペースに存在するポッド、サービス、ボリュームの数を制御することもできます。
+
+[Limit ranges](/docs/tasks/administer-cluster/manage-resources/memory-default-namespace/)は、上記のリソースの一部の最大または最小サイズを制限することで、ユーザーがメモリなどの一般的に予約されたリソースに対して不当に高いまたは低い値を要求するのを防いだり、何も指定されていない場合にデフォルトの制限を提供したりします。
+
+### コンテナが利用する特権の制御
+
+ポッドの定義には、[security context](/docs/tasks/configure-pod-container/security-context/)が含まれており、ノード上の特定の Linux ユーザー(root など)として実行するためのアクセス、特権的に実行するためのアクセス、ホストネットワークにアクセスするためのアクセス、その他の制御を要求することができます。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:90]:
```suggestion
ポッドの定義には、[security context](/docs/tasks/configure-pod-container/security-context/)が含まれており、ノード上の特定の Linux ユーザー(rootなど)として実行するためのアクセス、特権的に実行するためのアクセス、ホストネットワークにアクセスするためのアクセス、その他の制御を要求することができます。
```

File: content/ja/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 6]
```diff
@@ -0,0 +1,206 @@
...
+
+名前空間の[network policies](/docs/tasks/administer-cluster/declare-network-policy/)により、アプリケーション作成者は、他の名前空間のポッドが自分の名前空間内のポッドやポートにアクセスすることを制限することができます。
+
+サポートされている[Kubernetes networking providers](/docs/concepts/cluster-administration/networking/)の多くは、ネットワークポリシーを尊重するようになりました。
+クォータやリミットの範囲は、ユーザーがノードポートや負荷分散サービスを要求するかどうかを制御するためにも使用でき、多くのクラスターでは、ユーザーのアプリケーションがクラスターの外で見えるかどうかを制御できます。
+ノードごとのファイアウォール、クロストークを防ぐための物理的なクラスタノードの分離、高度なネットワークポリシーなど、プラグインや環境ごとにネットワークルールを制御する追加の保護機能が利用できる場合もあります。
+
+### クラウドメタデータのAPIアクセスを制限
+
+クラウドプラットフォーム（AWS、Azure、GCEなど）では、しばしばメタデータサービスをインスタンスにローカルに公開しています。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:128]:
```suggestion
クラウドプラットフォーム(AWS、Azure、GCEなど)では、しばしばメタデータサービスをインスタンスにローカルに公開しています。
```

File: content/ja/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 7]
```diff
@@ -0,0 +1,206 @@
...
+管理者としては、ベータ版のアドミッションプラグイン「PodNodeSelector」を使用して、ネームスペース内のポッドをデフォルトまたは特定のノードセレクタを必要とするように強制することができます。
+エンドユーザーがネームスペースを変更できない場合は、特定のワークロード内のすべてのポッドの配置を強く制限することができます。
+
+## クラスタのコンポーネントの保護
+
+このセクションでは、クラスターを危険から守るための一般的なパターンを説明します。
+
+### etcdへのアクセスの制限
+
+API 用の etcd バックエンドへの書き込みアクセスは、クラスタ全体の root を取得するのと同等であり、読み取りアクセスはかなり迅速にエスカレートするために使用できます。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:149]:
```suggestion
API用のetcdバックエンドへの書き込みアクセスは、クラスタ全体のrootを取得するのと同等であり、読み取りアクセスはかなり迅速にエスカレートするために使用できます。
```

Comment (nasa9084) [reply content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:149]: 以下同様

File: content/ja/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 8]
```diff
@@ -0,0 +1,206 @@
...
+Kubernetesは1.7で導入された機能である[encryption at rest](/docs/tasks/administer-cluster/encrypt-data/)をサポートしており、1.13からはベータ版となっています。
+これは、etcdの`Secret`リソースを暗号化し、etcdのバックアップにアクセスした人が、それらのシークレットの内容を見ることを防ぎます。
+この機能は現在ベータ版ですが、バックアップが暗号化されていない場合や、攻撃者が etcd への読み取りアクセスを得た場合に、追加の防御レベルを提供します。
+
+### セキュリティアップデートのアラートの受信と脆弱性の報告
+
+Join the [kubernetes-announce](https://groups.google.com/forum/#!forum/kubernetes-announce)
+group for emails about security announcements. See the
+[security reporting](/docs/reference/issues-security/security/)
+page for more on how to report vulnerabilities.
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:200]: このブロック消し忘れですね

**Review by nasa9084 — CHANGES_REQUESTED** [review]

File: content/ja/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 9]
```diff
@@ -0,0 +1,198 @@
...
+
+コンポーネントやインストール方法によっては、HTTP上のローカルポートを有効にする場合があることに注意してください。管理者は、潜在的に保護されていないトラフィックを特定するために、各コンポーネントの設定に精通している必要があります。
+
+### APIの認証
+
+クラスタのインストール時に、共通のアクセスパターンに合わせて、APIサーバーが使用する認証メカニズムを選択します。
+例えば、シングルユーザーの小規模なクラスターでは、シンプルな証明書や静的なBearerトークンを使用することができます。
+大規模なクラスターでは、ユーザーをグループに細分化できる既存のOIDCまたはLDAPサーバーを統合することができます。
+
+ノード、プロキシ、スケジューラ、ボリュームプラグインなど、インフラの一部であるものも含めて、すべてのAPIクライアントを認証する必要があります。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:37]:
```suggestion
ノード、プロキシー、スケジューラー、ボリュームプラグインなど、インフラの一部であるものも含めて、すべてのAPIクライアントを認証する必要があります。
```

File: content/ja/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 10]
```diff
@@ -0,0 +1,198 @@
...
+これらのクライアントは通常、[service accounts](/docs/reference/access-authn-authz/service-accounts-admin/)であるか、またはx509クライアント証明書を使用しており、クラスタ起動時に自動的に作成されるか、クラスタインストールの一部として設定されます。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:38]:
```suggestion
これらのクライアントは通常、[service accounts](/docs/reference/access-authn-authz/service-accounts-admin/)であるか、またはx509クライアント証明書を使用しており、クラスター起動時に自動的に作成されるか、クラスターインストールの一部として設定されます。
```

File: content/ja/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 11]
```diff
@@ -0,0 +1,198 @@
...
+
+### すべてのAPIトラフィックにTLS(Transport Layer Security)の使用
+
+Kubernetesは、クラスタ内のすべてのAPI通信がデフォルトでTLSにより暗号化されていることを期待しており、大半のインストール方法では、必要な証明書を作成してクラスタコンポーネントに配布することができます。
+
+コンポーネントやインストール方法によっては、HTTP上のローカルポートを有効にする場合があることに注意してください。管理者は、潜在的に保護されていないトラフィックを特定するために、各コンポーネントの設定に精通している必要があります。
+
+### APIの認証
+
+クラスタのインストール時に、共通のアクセスパターンに合わせて、APIサーバーが使用する認証メカニズムを選択します。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:33]:
```suggestion
クラスターのインストール時に、共通のアクセスパターンに合わせて、APIサーバーが使用する認証メカニズムを選択します。
```

File: content/ja/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 12]
```diff
@@ -0,0 +1,198 @@
...
+
+* {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+
+
+<!-- steps -->
+
+## Kubernetes APIへのアクセスの制御
+
+Kubernetesは完全にAPI駆動であるため、誰がクラスタにアクセスできるか、どのようなアクションを実行できるかを制御・制限することが第一の防御策となります。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:23]:
```suggestion
Kubernetesは完全にAPI駆動であるため、誰がクラスターにアクセスできるか、どのようなアクションを実行できるかを制御・制限することが第一の防御策となります。
```

File: content/ja/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 13]
```diff
@@ -0,0 +1,198 @@
...
+
+### すべてのAPIトラフィックにTLS(Transport Layer Security)の使用
+
+Kubernetesは、クラスタ内のすべてのAPI通信がデフォルトでTLSにより暗号化されていることを期待しており、大半のインストール方法では、必要な証明書を作成してクラスタコンポーネントに配布することができます。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:27]:
```suggestion
Kubernetesは、クラスター内のすべてのAPI通信がデフォルトでTLSにより暗号化されていることを期待しており、大半のインストール方法では、必要な証明書を作成してクラスターコンポーネントに配布することができます。
```

File: content/ja/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 14]
```diff
@@ -0,0 +1,198 @@
...
+
+
+
+<!-- steps -->
+
+## Kubernetes APIへのアクセスの制御
+
+Kubernetesは完全にAPI駆動であるため、誰がクラスタにアクセスできるか、どのようなアクションを実行できるかを制御・制限することが第一の防御策となります。
+
+### すべてのAPIトラフィックにTLS(Transport Layer Security)の使用
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:25]:
```suggestion
### すべてのAPIトラフィックにTLS(Transport Layer Security)を使用する
```

File: content/ja/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 15]
```diff
@@ -0,0 +1,198 @@
...
+### APIの認証
+
+クラスタのインストール時に、共通のアクセスパターンに合わせて、APIサーバーが使用する認証メカニズムを選択します。
+例えば、シングルユーザーの小規模なクラスターでは、シンプルな証明書や静的なBearerトークンを使用することができます。
+大規模なクラスターでは、ユーザーをグループに細分化できる既存のOIDCまたはLDAPサーバーを統合することができます。
+
+ノード、プロキシ、スケジューラ、ボリュームプラグインなど、インフラの一部であるものも含めて、すべてのAPIクライアントを認証する必要があります。
+これらのクライアントは通常、[service accounts](/docs/reference/access-authn-authz/service-accounts-admin/)であるか、またはx509クライアント証明書を使用しており、クラスタ起動時に自動的に作成されるか、クラスタインストールの一部として設定されます。
+
+詳細については、[authentication reference document](/docs/reference/access-authn-authz/authentication/)を参照してください。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:40]: The page has been translated, please update link text and url

File: content/ja/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 16]
```diff
@@ -0,0 +1,198 @@
...
+
+### APIの認可
+
+認証されると、すべてのAPIコールは認可チェックを通過することになります。
+
+Kubernetesには、統合された[Role-Based Access Control (RBAC)](/docs/reference/access-authn-authz/rbac/)コンポーネントが搭載されており、入力されたユーザーやグループを、ロールにまとめられたパーミッションのセットにマッチさせます。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:46]: The page has been translated, please update link text and url

Comment (nasa9084) [reply content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:46]: please check below links too

File: content/ja/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 17]
```diff
@@ -0,0 +1,198 @@
...
+これらのパーミッションは、動詞(get, create, delete)とリソース(pods, services, nodes)を組み合わせたもので、ネームスペース・スコープまたはクラスター・スコープに対応しています。
+すぐに使えるロールのセットが提供されており、クライアントが実行したいアクションに応じて、デフォルトで適切な責任の分離を提供します。
+
+[Node](/docs/reference/access-authn-authz/node/)と[RBAC](/docs/reference/access-authn-authz/rbac/)の承認者は、[NodeRestriction](/docs/reference/access-authn-authz/admission-controllers/#noderestriction)のアドミッションプラグインと組み合わせて使用することをお勧めします。
+
+認証の場合と同様に、小規模なクラスタにはシンプルで幅広い役割が適切かもしれません。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:52]:
```suggestion
認証の場合と同様に、小規模なクラスターにはシンプルで幅広い役割が適切かもしれません。
```

please fix below too

**Review by nasa9084 — CHANGES_REQUESTED** [review]
I have reviewed above L70

File: content/ja/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 18]
```diff
@@ -0,0 +1,198 @@
...
+
+詳細は、[Kubelet 認証/認可](/ja/docs/reference/command-line-tools-reference/kubelet-authentication-authorization)に参照してください。
+
+## ワークロードやユーザーのキャパシティーを実行時に制御
+
+Kubernetesにおける権限付与は、意図的にハイレベルであり、リソースに対する粗いアクションに焦点を当てています。
+
+より強力なコントロールは**policies**として存在し、それらのオブジェクトがクラスタや自身、その他のリソースにどのように作用するかをユースケースによって制限します。
+
+### クラスタのリソース使用量の制限
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:78]:
```suggestion
### クラスターのリソース使用量の制限
```

File: content/ja/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 19]
```diff
@@ -0,0 +1,198 @@
...
+
+[リソースクォータ](/ja/docs/concepts/policy/resource-quotas/)は、ネームスペースに付与されるリソースの数や容量を制限するものです。
+
+これは、ネームスペースが割り当てることのできるCPU、メモリ、永続的なディスクの量を制限するためによく使われますが、各ネームスペースに存在するPod、サービス、ボリュームの数を制御することもできます。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:82]:
```suggestion
これは、ネームスペースが割り当てることのできるCPU、メモリー、永続的なディスクの量を制限するためによく使われますが、各ネームスペースに存在するPod、サービス、ボリュームの数を制御することもできます。
```

File: content/ja/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 20]
```diff
@@ -0,0 +1,198 @@
...
+
+[Limit ranges](/docs/tasks/administer-cluster/manage-resources/memory-default-namespace/)は、上記のリソースの一部の最大または最小サイズを制限することで、ユーザーがメモリなどの一般的に予約されたリソースに対して不当に高いまたは低い値を要求するのを防いだり、何も指定されていない場合にデフォルトの制限を提供したりします。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:84]:
```suggestion
[Limit ranges](/docs/tasks/administer-cluster/manage-resources/memory-default-namespace/)は、上記のリソースの一部の最大または最小サイズを制限することで、ユーザーがメモリーなどの一般的に予約されたリソースに対して不当に高いまたは低い値を要求するのを防いだり、何も指定されていない場合にデフォルトの制限を提供したりします。
```

File: content/ja/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 21]
```diff
@@ -0,0 +1,198 @@
...
+
+### コンテナが利用する特権の制御
+
+Podの定義には、[security context](/docs/tasks/configure-pod-container/security-context/)が含まれており、ノード上の特定の Linux ユーザー(rootなど)として実行するためのアクセス、特権的に実行するためのアクセス、ホストネットワークにアクセスするためのアクセス、その他の制御を要求することができます。
+[Pod security policies](/docs/concepts/policy/pod-security-policy/)は、危険なセキュリティコンテキスト設定を提供できるユーザーやサービスアカウントを制限することができます。
+
+たとえば、Podのセキュリティポリシーでは、ボリュームマウント、特に`hostPath`を制限することができ、これはPodの制御すべき側面です。
+一般に、ほとんどのアプリケーションワークロードでは、ホストリソースへのアクセスを制限する必要があります。
+ホスト情報にアクセスすることなく、ルートプロセス（uid 0）として正常に実行できます。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:93]:
```suggestion
ホスト情報にアクセスすることなく、ルートプロセス(uid 0)として正常に実行できます。
```

File: content/ja/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 22]
```diff
@@ -0,0 +1,198 @@
...
+
+名前空間の[ネットワークポリシー](/ja/docs/tasks/administer-cluster/declare-network-policy/)により、アプリケーション作成者は、他の名前空間のPodが自分の名前空間内のPodやポートにアクセスすることを制限することができます。
+
+サポートされている[Kubernetes networking providers](/ja/docs/concepts/cluster-administration/networking/)の多くは、ネットワークポリシーを尊重するようになりました。
+クォータやリミットの範囲は、ユーザーがノードポートや負荷分散サービスを要求するかどうかを制御するためにも使用でき、多くのクラスターでは、ユーザーのアプリケーションがクラスターの外で見えるかどうかを制御できます。
+ノードごとのファイアウォール、クロストークを防ぐための物理的なクラスタノードの分離、高度なネットワークポリシーなど、プラグインや環境ごとにネットワークルールを制御する追加の保護機能が利用できる場合もあります。
+
+### クラウドメタデータのAPIアクセスを制限
+
+クラウドプラットフォーム(AWS、Azure、GCEなど)では、しばしばメタデータサービスをインスタンスにローカルに公開しています。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:126]:
```suggestion
クラウドプラットフォーム(AWS、Azure、GCEなど)では、しばしばメタデータサービスをインスタンスローカルに公開しています。
```

File: content/ja/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 23]
```diff
@@ -0,0 +1,198 @@
...
+デフォルトでは、これらのAPIはインスタンス上で実行されているPodからアクセスでき、そのノードのクラウド認証情報や、kubelet認証情報などのプロビジョニングデータを含むことができます。
+これらの認証情報は、クラスタ内でのエスカレーションや、同じアカウントの他のクラウドサービスへのエスカレーションに使用できます。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:128]:
```suggestion
これらの認証情報は、クラスター内でのエスカレーションや、同じアカウントの他のクラウドサービスへのエスカレーションに使用できます。
```

File: content/ja/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 24]
```diff
@@ -0,0 +1,198 @@
...
+### Podのアクセス可能ノードを制御
+
+デフォルトでは、どのノードがPodを実行できるかについての制限はありません。
+Kubernetesは、エンドユーザーが利用できる[Node上へのPodのスケジューリング](/ja/docs/concepts/scheduling-eviction/assign-pod-node/)と[TaintとToleration](/ja/docs/concepts/scheduling-eviction/taint-and-toleration/)を提供します。
+多くのクラスターでは、ワークロードを分離するためにこれらのポリシーを使用することは、作者が採用したり、ツールを使って強制したりする慣習になっています。
+
+管理者としては、ベータ版のアドミッションプラグイン「PodNodeSelector」を使用して、ネームスペース内のPodをデフォルトまたは特定のノードセレクタを必要とするように強制することができます。
+エンドユーザーがネームスペースを変更できない場合は、特定のワークロード内のすべてのPodの配置を強く制限することができます。
+
+## クラスタのコンポーネントの保護
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:141]:
```suggestion
## クラスターのコンポーネントの保護
```

File: content/ja/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 25]
```diff
@@ -0,0 +1,198 @@
...
+
+このセクションでは、クラスターを危険から守るための一般的なパターンを説明します。
+
+### etcdへのアクセスの制限
+
+API用のetcdバックエンドへの書き込みアクセスは、クラスタ全体のrootを取得するのと同等であり、読み取りアクセスはかなり迅速にエスカレートするために使用できます。
+管理者は、TLSクライアント証明書による相互認証など、APIサーバーからetcdサーバーへの強力な認証情報を常に使用すべきであり、API サーバーのみがアクセスできるファイアウォールの後ろにetcdサーバーを隔離することがしばしば推奨される。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:148]:
```suggestion
管理者は、TLSクライアント証明書による相互認証など、APIサーバーからetcdサーバーへの強力な認証情報を常に使用すべきであり、API サーバーのみがアクセスできるファイアウォールの後ろにetcdサーバーを隔離することがしばしば推奨されます。
```

File: content/ja/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 26]
```diff
@@ -0,0 +1,198 @@
...
+
+{{< caution >}}
+クラスタ内の他のコンポーネントが、完全なキースペースへの読み取りまたは書き込みアクセスを持つマスターetcdインスタンスへのアクセスを許可することは、クラスタ管理者のアクセスを許可することと同じです。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:151]:
```suggestion
クラスター内の他のコンポーネントが、完全なキースペースへの読み取りまたは書き込みアクセスを持つマスターetcdインスタンスへのアクセスを許可することは、クラスター管理者のアクセスを許可することと同じです。
```

File: content/ja/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 27]
```diff
@@ -0,0 +1,198 @@
...
+
+アルファ版およびベータ版のKubernetesの機能は活発に開発が行われており、セキュリティ上の脆弱性をもたらす制限やバグがある可能性があります。
+常に、アルファ版またはベータ版の機能が提供する価値と、セキュリティ体制に起こりうるリスクを比較して評価してください。
+疑問がある場合は、使用しない機能を無効にしてください。
+
+### インフラの認証情報を頻繁に交換
+
+秘密やクレデンシャルの有効期間が短いほど、攻撃者がそのクレデンシャルを利用することは難しくなります。
+証明書の有効期間を短く設定し、そのローテーションを自動化します。
+発行されたトークンの利用可能期間を制御できる認証プロバイダを使用し、可能な限り短いライフタイムを使用します。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:171]:
```suggestion
発行されたトークンの利用可能期間を制御できる認証プロバイダーを使用し、可能な限り短いライフタイムを使用します。
```

File: content/ja/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 28]
```diff
@@ -0,0 +1,198 @@
...
+外部統合でサービス・アカウント・トークンを使用する場合、これらのトークンを頻繁にローテーションすることを計画します。
+例えば、ブートストラップ・フェーズが完了したら、ノードのセットアップに使用したブートストラップ・トークンを失効させるか、その認証を解除する必要があります。
+
+### サードパーティの統合を有効にする前に確認
+
+Kubernetesへの多くのサードパーティの統合は、クラスタのセキュリティプロファイルを変更する可能性があります。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:177]:
```suggestion
Kubernetesへの多くのサードパーティの統合は、クラスターのセキュリティプロファイルを変更する可能性があります。
```

File: content/ja/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 29]
```diff
@@ -0,0 +1,198 @@
...
+統合を有効にする際には、アクセスを許可する前に、拡張機能が要求するパーミッションを常に確認してください。
+
+例えば、多くのセキュリティ統合は、事実上そのコンポーネントをクラスタ管理者にしているクラスタ上のすべての秘密を見るためのアクセスを要求するかもしれません。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:180]:
```suggestion
例えば、多くのセキュリティ統合は、事実上そのコンポーネントをクラスター管理者にしているクラスター上のすべての秘密を見るためのアクセスを要求するかもしれません。
```

File: content/ja/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 30]
```diff
@@ -0,0 +1,198 @@
...
+疑問がある場合は、可能な限り単一の名前空間で機能するように統合を制限してください。
+Podを作成するコンポーネントも、`kube-system`名前空間のような名前空間内で行うことができれば、予想外に強力になる可能性があります。これは、サービスアカウントのシークレットにアクセスしたり、サービスアカウントに寛容な[pod security policies](/docs/concepts/policy/pod-security-policy/)へのアクセスが許可されている場合に、昇格したパーミッションでPodが実行される可能性があるからです。
+
+### etcdにあるSecretを暗号化
+
+一般的に、etcdデータベースにはKubernetes APIを介してアクセス可能なあらゆる情報が含まれており、攻撃者はクラスタの状態に大きな可視性を与える可能性があります。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:186]:
```suggestion
一般的に、etcdデータベースにはKubernetes APIを介してアクセス可能なあらゆる情報が含まれており、クラスターの状態に対する大きな可視性を攻撃者へ与える可能性があります。
```

File: content/ja/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 31]
```diff
@@ -0,0 +1,198 @@
...
+よく吟味されたバックアップおよび暗号化ソリューションを使用して、常にバックアップを暗号化し、可能な場合はフルディスク暗号化の使用を検討してください。
+
+Kubernetesは1.7で導入された機能である[encryption at rest](/docs/tasks/administer-cluster/encrypt-data/)をサポートしており、1.13からはベータ版となっています。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:189]:
```suggestion
Kubernetesは1.7で導入された機能である[encryption at rest](/docs/tasks/administer-cluster/encrypt-data/)をサポートしており、これは1.13からはベータ版となっています。
```

File: content/ja/docs/tasks/administer-cluster/securing-a-cluster.md [hunk 32]
```diff
@@ -0,0 +1,198 @@
...
+これは、etcdの`Secret`リソースを暗号化し、etcdのバックアップにアクセスした人が、それらのシークレットの内容を見ることを防ぎます。
+この機能は現在ベータ版ですが、バックアップが暗号化されていない場合や、攻撃者がetcdへの読み取りアクセスを得た場合に、追加の防御レベルを提供します。
+
+### セキュリティアップデートのアラートの受信と脆弱性の報告
+
+[kubernetes-announce](https://groups.google.com/forum/#!forum/kubernetes-announce)に参加してください。
+グループに参加すると、セキュリティアナウンスに関するメールを受け取ることができます。
+脆弱性の報告方法については、[security reporting](/docs/reference/issues-security/security/)
+ページに参照してください。
```
Comment (nasa9084) [inline content/ja/docs/tasks/administer-cluster/securing-a-cluster.md:198]:
```suggestion
脆弱性の報告方法については、[security reporting](/docs/reference/issues-security/security/)ページを参照してください。
```

**Review by nasa9084 — CHANGES_REQUESTED** [review]

**Review by nasa9084 — COMMENTED** [review]
/approve

**Comment by ptux** [comment]
@inductor 

Cloud you take a look at this PR?

**Pull request merged**